microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls.md

```diff
@@ -1,162 +1,49 @@
 ---
-title: Handle URLs in Progressive Web Apps
-description: How to register your PWA as a URL handler to more deeply integrate it in the operating system with other applications.
+title: Handle links to your Progressive Web Apps
+description: How links to your PWA can be handled by your app rather than the web browser.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: pwa
-ms.date: 01/12/2022
+ms.date: 09/15/2022
 ---
-# Handle URLs in Progressive Web Apps
+# Handle links to your Progressive Web Apps
 
-Native applications on many operating systems can be associated with URLs. They can request to be launched, instead of the browser, when associated URLs are activated.
+Native applications on many operating systems can handle links. They can request to be launched, instead of the web browser, when associated URLs are activated and doing so creates a more engaging experience for these applications.
```

Comment:
```suggestion
Native applications on many operating systems can handle links. Native applications can request to be launched, instead of the web browser, when associated URLs are activated, to create a more engaging experience for these applications.
```
It was unclear to me whether 'they' meant links or native apps.